Symfony/CS/Tests/Fixer/Symfony/PhpdocInlineTagFixerTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Symfony;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * Test PhpdocInlineTagFixer.
+ */
+class PhpdocInlineTagFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideTestFixInlineDocCases
+     */
+    public function testFixInlineDoc($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideTestFixInlineDocCases()
+    {
+        $cases = array(
+            array(
+                '<?php
+    /**
+     * { test }
+     * {@inheritdoc test}
+     * test other comment
+     * {@inheritdoc test} a
+     * {@inheritdoc test} b
+     * {@inheritdoc test} c
+     * {@inheritdoc foo bar.} d
+     * {@inheritdoc foo bar.} e
+     * {@inheritdoc test} f
+     * end comment {@inheritdoc here we are done} @spacepossum {1}
+     */
+',
+                '<?php
+    /**
+     * { test }
```

Comment:
please add something like `{ link }`. 
What is the expected result?

At the moment it is (I guess): `{@link}`